ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -229,12 +233,20 @@ private void validateConfig(final JsonRpcConfiguration config) {
                 LOG.info(
                     "JsonRPC service started and listening on {}:{}", config.getHost(), actualPort);
                 config.setPort(actualPort);
-                // Request that a NAT port forward for our server port
-                if (natManager.isPresent()) {
-                  natManager
-                      .get()
-                      .requestPortForward(
-                          config.getPort(), UpnpNatManager.Protocol.TCP, "partheon-json-rpc");
+
+                if (natService.isPresent()) {
+                  final NatService service = natService.get();
+                  // request that a NAT port forward for our server port (if UPNP Nat Environment)
+                  if (service.isNatEnvironment() && service.getNatMethod().equals(NatMethod.UPNP)) {
+                    service
+                        .getNatManager()
+                        .ifPresent(
+                            natSystem -> {
+                              final UpnpNatManager upnpNatSystem = (UpnpNatManager) natSystem;
+                              upnpNatSystem.requestPortForward(
+                                  config.getPort(), NetworkProtocol.TCP, NatServiceType.JSON_RPC);
+                            });
+                  }
```

Comment:
Since there are a few `Optional`s in here, let's try to use `filter`, `flatMap`, `map`, `ifPresent`, etc. In place of mixing `if`s and `get`s. I think this can be reduced to one chain starting with `natService` :)

Reply:
I just changed as you requested and indeed the result is better :)